tests/mu_pri_enum_test.c

```diff
@@ -50,18 +50,18 @@ static int verifyMU_ENUM_VALUE_BC_plus_1(const char* empty, const char* valueAsS
 }
 
 #define TEST_PRI_ENUM_2_VALUES    \
-    X2, 2,                        \
-    ZY2, 6
+    X2 = 2,                        \
+    ZY2 = 6
 
-MU_DEFINE_ENUM_2(TEST_PRI_ENUM_2, TEST_PRI_ENUM_2_VALUES);
-MU_DEFINE_ENUM_STRINGS_2(TEST_PRI_ENUM_2, TEST_PRI_ENUM_2_VALUES);
+MU_DEFINE_ENUM(TEST_PRI_ENUM_2, TEST_PRI_ENUM_2_VALUES);
+MU_DEFINE_ENUM_STRINGS(TEST_PRI_ENUM_2, TEST_PRI_ENUM_2_VALUES);
```

Comment:
I would love to see here some tests that take an enum having implicit and explicit values, such as 

enum E_TAG
{
    e,
    f=e,
    g=f+1,
    h,
    i
}

and an enum whose first value is explicit

enum F_TAG
{ 
     f=6,
     g,
     h=34
}